### PR TITLE
replace ls with exa

### PR DESCRIPTION
**Pull request by ChrisB9**

### CODE REVIEW

**Review by Kanti — APPROVED** [review]

**Comment by Cyb10101** [comment]
Maybe you don't want to overwrite LS_COLORS:

```bash
echo $LS_COLORS

rs=0:di=01;34:ln=01;36:mh=00:pi=40;33:so=01;35:do=01;35:bd=40;33;01:cd=40;33;01:or=40;31;01:mi=00:su=37;41:sg=30;43:ca=30;41:tw=30;42:ow=34;42:st=37;44:ex=01;32:*.tar=01;31:*.tgz=01;31:*.arc=01;31:*.arj=01;31:*.taz=01;31:*.lha=01;31:*.lz4=01;31:*.lzh=01;31:*.lzma=01;31:*.tlz=01;31:*.txz=01;31:*.tzo=01;31:*.t7z=01;31:*.zip=01;31:*.z=01;31:*.dz=01;31:*.gz=01;31:*.lrz=01;31:*.lz=01;31:*.lzo=01;31:*.xz=01;31:*.zst=01;31:*.tzst=01;31:*.bz2=01;31:*.bz=01;31:*.tbz=01;31:*.tbz2=01;31:*.tz=01;31:*.deb=01;31:*.rpm=01;31:*.jar=01;31:*.war=01;31:*.ear=01;31:*.sar=01;31:*.rar=01;31:*.alz=01;31:*.ace=01;31:*.zoo=01;31:*.cpio=01;31:*.7z=01;31:*.rz=01;31:*.cab=01;31:*.wim=01;31:*.swm=01;31:*.dwm=01;31:*.esd=01;31:*.jpg=01;35:*.jpeg=01;35:*.mjpg=01;35:*.mjpeg=01;35:*.gif=01;35:*.bmp=01;35:*.pbm=01;35:*.pgm=01;35:*.ppm=01;35:*.tga=01;35:*.xbm=01;35:*.xpm=01;35:*.tif=01;35:*.tiff=01;35:*.png=01;35:*.svg=01;35:*.svgz=01;35:*.mng=01;35:*.pcx=01;35:*.mov=01;35:*.mpg=01;35:*.mpeg=01;35:*.m2v=01;35:*.mkv=01;35:*.webm=01;35:*.ogm=01;35:*.mp4=01;35:*.m4v=01;35:*.mp4v=01;35:*.vob=01;35:*.qt=01;35:*.nuv=01;35:*.wmv=01;35:*.asf=01;35:*.rm=01;35:*.rmvb=01;35:*.flc=01;35:*.avi=01;35:*.fli=01;35:*.flv=01;35:*.gl=01;35:*.dl=01;35:*.xcf=01;35:*.xwd=01;35:*.yuv=01;35:*.cgm=01;35:*.emf=01;35:*.ogv=01;35:*.ogx=01;35:*.aac=00;36:*.au=00;36:*.flac=00;36:*.m4a=00;36:*.mid=00;36:*.midi=00;36:*.mka=00;36:*.mp3=00;36:*.mpc=00;36:*.ogg=00;36:*.ra=00;36:*.wav=00;36:*.oga=00;36:*.opus=00;36:*.spx=00;36:*.xspf=00;36:
```

Maybe you want something more:

```bash
# grep
alias grep='grep --color=auto'
alias egrep='egrep --color=auto'
alias fgrep='fgrep --color=auto'

# List directory
alias l='ls -vC --color=auto --group-directories-first'
alias la='ls -vA --color=auto --group-directories-first'
alias ll='ls -vahl --color=auto --group-directories-first'
alias ls='ls -v --color=auto --group-directories-first'

# List directory https://the.exa.website/
if [ -x /usr/bin/exa ]; then
    alias l='exa --group-directories-first'
    alias la='exa -a --group-directories-first'
    alias ll='exa -ahl --git --header --group --group-directories-first'
    alias ls='exa --group-directories-first'
fi

# Change directory
alias ..='cd ..'
alias cd..='cd ..'

# Get current unixtime
alias unixtime='date +"%s"'

# Show all open ports
alias ports='netstat -tulanp'

# less
alias less='less -FSRX'
```

**Comment by ChrisB9** [comment]
while i definitively like some of the aliases you propose, those are a different topic.

Why should the binary check be necessary if exa is installed all the time?
I also don't think, that redefining the aliases is needed. 

I do agree with you on the other part of overriding LS_COLORS, we should rather append instead of setting it

**Comment by Cyb10101** [comment]
> Why should the binary check be necessary if exa is installed all the time? I also don't think, that redefining the aliases is needed.

Actually only if exa cannot be installed or is uninstalled.
That can only happen if you want to do gruesome things on multiple systems.
So that it doesn't collapse completely, I simply built in a small switch.
**In the case of a Docker image, that will never happen. Overwriting these variables is useless.**
I'm sometimes just a little sure and lazy at the same time. :-)

**Review by Kanti — APPROVED** [review]

File: .additional_bashrc.sh [hunk 1]
```diff
@@ -1,4 +1,7 @@
-alias ll='ls -alh'
+export LS_COLORS="di=1;34"
```
Comment (Kanti) [inline .additional_bashrc.sh:1]:
```suggestion
export LS_COLORS="$LS_COLORSdi=1;34:"
```

**Review by Kanti — APPROVED** [review]

**Pull request merged**